src/flows/context/flow.list.tsx

```diff
@@ -181,16 +219,27 @@ export const FlowListProvider: FC = ({children}) => {
       name: flow.name,
       range: flow.range,
       refresh: flow.refresh,
-      data: flow.data.serialize(),
-      meta: flow.meta.serialize(),
+      data: flow.data.serialize ? flow.data.serialize() : flow.data,
```

Comment:
didn't get to the bottom of this eh? for the life of me, i cant see why this condition shows up, which means it's prolly a reference error somewhere

Reply:
yeah I tried to hunt it down. You can reproduce it by creating a new Flow with this feature flag turned on. Figured this solution will work for now and we can try and solve it down the road